examples/complex/Main.lhs

```diff
@@ -218,7 +219,8 @@ prepare_configuration = do
 
     -- CM.setForwardTo c (Just $ RemotePipe "logs/pipe")
     -- CM.setForwardTo c (Just $ RemotePipe "\\\\.\\pipe\\acceptor") -- on Windows
-    -- CM.setForwardTo c (Just $ RemoteSocket "127.0.0.1" "2999")
+    CM.setForwardTo c (Just $ RemoteSocket "127.0.0.1" "42999")
```

Comment:
Are we ok with this being the default in `example-complex`?

Reply:
yes, for the time being..